src/main/java/seedu/address/model/person/Description.java

```diff
@@ -0,0 +1,72 @@
+package seedu.address.model.person;
+
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.util.Objects;
+
+
+/**
+ * Represents a Person's description in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidDescription(String)}
+ */
+public class Description {
+    public static final String MESSAGE_CONSTRAINTS = "Description can take any values, "
+            + "and it should not be empty if the /d flag has been input by user";
+
+    /*
+     * The first character of the description must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[^\\s].*";
```

Comment:
But if it can be any value, why are you using a validation regex? Shouldn't the check just be for whether or not String is empty or trivial (only spaces)?

Reply:
Is it possible that we want to prevent the use of certain special characters?

Reply:
My mistake, no problem!